src/maxdiffusion/generate_wan.py

```diff
@@ -141,6 +141,11 @@ def inference_generate_video(config, pipeline, filename_prefix=""):
 def run(config, pipeline=None, filename_prefix=""):
   print("seed: ", config.seed)
   model_key = config.model_name
+  tensorboard_dir = os.path.join(config.output_dir, "tensorboard")
```

Comment:
No need to set this, it is already initialized [here](https://github.com/AI-Hypercomputer/maxdiffusion/blob/main/src/maxdiffusion/pyconfig.py#L174) just use config.tensorboard_dir